docs/how_to_guides/how_to_setup_simple_chemistry.rst

```diff
@@ -461,3 +459,10 @@ In the example code above, we show how to setup the thermo and reaction packages
 and place them into the `EquilibriumReactor` unit model, but do not go further.
 Additional instructions for setting up and solving unit models can be found at
 `IDAESWorkflow`_.
+
+.. testoutput::
+   :hide:
+   :options: +ELLIPSIS
+
+   WARNING: DEPRECATED: The default argument for the ProcessBlock class is
+       ...

```

Comment:
I'm guessing this tests that the deprecation warning are hidden, and this links the use of an ellipsis to what should be hidden? Just curious to confirm how this works given the unfamiliar (to me) syntax.

Reply:
Oh, the ellipsis is a wildcard actually. Also, posting these links for anyone else who's curious:
https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html#directive-testoutput
https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html#configuration

Reply:
Should we try to fix this particular deprecation warning? We'll probably need to soon regardless.

Reply:
The deprecation warnings don't play well with doctest. I had to add the warning to the testoutput to get the test to pass. I can fix this in a future pull request after these warnings get fixed. But for now I think this is fine.